plugins/modules/cloudfront_info.py

```diff
@@ -494,7 +494,7 @@ def get_distribution_id_from_domain_name(self, domain_name):
         try:
             distribution_id = ""
             distributions = self.list_distributions(False)
-            distributions += self.list_streaming_distributions(False)
+            distributions + self.list_streaming_distributions(False)
```

Comment:
This looks like the assignment has been accidentally dropped?

Reply:
already fixed :)